Liotad autostart branch

ApurvaH opened the pull request:
Autostarts liota daemon in background.

KohliDev commented on scripts/autostartliota:
```diff
@@ -0,0 +1,32 @@
+	sudo python /etc/liota/packages/liotad.py &
+}
+
+case "$1" in
```
How the re-loading of loaded packages is taken care, in case of restart scenario?

KohliDev commented on setup.py:
```diff
@@ -123,5 +123,7 @@
                  ['config/liota.conf', 'config/logging.json']),
                 (os.path.abspath(os.sep) + '/../etc/liota',
                  ['BSD_LICENSE.txt', 'BSD_NOTICE.txt']),
+                (os.path.abspath(os.sep) + '/../etc/init.d',
```
Don`t copy forcefully in /etc/init.d, make it a manual step and document in README.md if user wants to use this functionality.

ApurvaH commented:
Hi Vaibhav,

There is a text file: liota/packages/package_auto.txt where you have to list the packages that you want to be loaded everytime.

I implemented the same thing in different manner. All the packages that you load are automatically recorded and the list is then used to load them at restart.
https://github.com/vmware/liota/compare/master...ApurvaH:autostart_branch

Since this functionality has already been implemented I removed it from the pull request.

Thanks,
Apurva

From: Vaibhav Kohli [mailto:notifications@github.com]
Sent: Wednesday, February 1, 2017 11:37 AM
To: vmware/liota <liota@noreply.github.com>
Cc: Apurva Hajare <ahajare@vmware.com>; Author <author@noreply.github.com>
Subject: Re: [vmware/liota] Liotad autostart branch (#76)


@KohliDev commented on this pull request.

________________________________

In scripts/autostartliota<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_liota_pull_76-23pullrequestreview-2D19487298&d=DwMCaQ&c=uilaK90D4TOVoH58JNXRgQ&r=HOT7utYnlQOW0a3RCyEnWVrL5BPCZzKgDDPhZlzZ4Lo&m=fd0oZc2PbMN1yN_MicVUtuXwYhXWpVMregxKtKZLcTg&s=H_3mLI7g7XH5kgzbK0MfTbFGWqs4skE5_dFxgPF8r_8&e=>:

> +### BEGIN INIT INFO

+# Provides:          Autostarts liota.

+# Required-Start:    $all

+# Required-Stop:

+# Default-Start:     2 3 4 5

+# Default-Stop:

+# Short-Description: Starts liotad.py in background at reboot.

+### END INIT INFO

+

+PATH=/sbin:/usr/sbin:/bin:/usr/bin

+

+do_start() {

+       sudo python /etc/liota/packages/liotad.py &

+}

+

+case "$1" in

How the re-loading of loaded packages is taken care, in case of restart scenario?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_liota_pull_76-23pullrequestreview-2D19487298&d=DwMCaQ&c=uilaK90D4TOVoH58JNXRgQ&r=HOT7utYnlQOW0a3RCyEnWVrL5BPCZzKgDDPhZlzZ4Lo&m=fd0oZc2PbMN1yN_MicVUtuXwYhXWpVMregxKtKZLcTg&s=H_3mLI7g7XH5kgzbK0MfTbFGWqs4skE5_dFxgPF8r_8&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AWXIW9YEZFm6eOR59F24N103DEY-2Dfc-5FWks5rYCEJgaJpZM4Lyf4A&d=DwMCaQ&c=uilaK90D4TOVoH58JNXRgQ&r=HOT7utYnlQOW0a3RCyEnWVrL5BPCZzKgDDPhZlzZ4Lo&m=fd0oZc2PbMN1yN_MicVUtuXwYhXWpVMregxKtKZLcTg&s=R9hXiSQFVQPtJjym8m_Et4TIyyRRE6ClCf9PT5JVhK4&e=>.

pmasrani commented on scripts/autostartliota:
```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
```
Add the copyright header.

KohliDev commented on README.md:
```diff
@@ -54,6 +54,50 @@ In general, liota can be installed with:
+## Autostarting Liota Daemon
+For starting liotad.py in background automatically at reboot perform the following steps: 
+
+* Copy autostartliota script present in scripts folder to location:
```
What is the location of script folder ???
And I don`t see "scripts/autostartliota" included in setup.py in this PR to exist at "/etc/liota" location.
